python/mujoco/usd/shapes.py

```diff
@@ -14,34 +14,244 @@
 # ==============================================================================
 """Built-in shapes for USD exporter."""
 
-from typing import Dict, Any
+from typing import Dict, Any, Tuple, Optional, Union
 
 import mujoco
 import numpy as np
-from open3d import open3d as o3d
 
-
-def create_hemisphere(
-    radius: float, theta_steps: int = 50, phi_steps: int = 50
+def get_triangle_uvs(
+    vertices: np.array,
```

Comment:
The type is not np.array (which is a function). You can either use np.ndarray or use numpy.typing.ArrayLike